docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -231,15 +230,15 @@ yarn add @material-ui/styles@next
 
 ### ExpansionPanel
 
-- [ExpansionPanelActions] Rename the `action` CSS class `spacing`.
+- [ExpansionPanelActions] Rename the `action` CSS class to `spacing`.
 - [ExpansionPanel] Increase the CSS specificity of the `disabled` style rule.
 
 ### List
 
 - [List] Rework the list components to match the specification:
 
-  - The usage of the `ListItemAvatar` component is required when using an avatar
-  - The usage of the `ListItemIcon` component is required when using a left checkbox
+  - The use of the `ListItemAvatar` component is required when using an avatar`.
```

Comment:
Dropping ’The use of’ is probably better here

Reply:
Thanks Josh. Changes pushed :)